### PR TITLE
M60 simple 5x12 in lego keyboard

### PR DESCRIPTION
**Pull request by alinelena**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/mlego/m60/keymaps/default/keymap.c [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+bool encoder_update_user(uint8_t index, bool clockwise) {
+    uint16_t held_keycode_timer = timer_read();
+
+    if (index == 0) { /* First encoder */
+        if (clockwise) {
+            register_code(KC_VOLD);
+            while (timer_elapsed(held_keycode_timer) < MEDIA_KEY_DELAY) {
+                // no-op
+            }
+            unregister_code(KC_VOLD);
```
Comment (drashna) [inline keyboards/mlego/m60/keymaps/default/keymap.c:137]: This entire block can be replaced with simply: 
```suggestion
            tap_code(KC_VOLD, MEDIA_KEY_DELAY);
```

File: keyboards/mlego/m60/readme.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# M60
+
+![M60](https://i.imgur.com/Zm7LaHi.jpg)
```
Comment (fauxpark) [inline keyboards/mlego/m60/readme.md:3]:
```suggestion
![M60](https://i.imgur.com/Zm7LaHih.jpg)

```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/mlego/m60/m60.h [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_5x12(K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411) \
+    {                                                                                                                                                                                                                                                                                                                                                                                             \
+        {K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011}, {K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111}, {K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211}, {K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311}, { K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411 }        \
+    }
```
Comment (noroadsleft) [inline keyboards/mlego/m60/m60.h:25]: This is just a white space change, but not having the layout macro formatted somewhat defeats its purpose.

```suggestion
#define LAYOUT_ortho_5x12( \
    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, \
    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, \
    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, \
    K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, \
    K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411  \
) { \
    {K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011}, \
    {K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111}, \
    {K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211}, \
    {K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311}, \
    {K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411}  \
}
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/mlego/m60/m60.h (hunk 3 above)
Comment (noroadsleft) [inline keyboards/mlego/m60/m60.h:25]:
```suggestion
#define LAYOUT_ortho_5x12( \
    K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011, \
    K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111, \
    K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211, \
    K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311, \
    K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411  \
) { \
    {K000, K001, K002, K003, K004, K005, K006, K007, K008, K009, K010, K011}, \
    {K100, K101, K102, K103, K104, K105, K106, K107, K108, K109, K110, K111}, \
    {K200, K201, K202, K203, K204, K205, K206, K207, K208, K209, K210, K211}, \
    {K300, K301, K302, K303, K304, K305, K306, K307, K308, K309, K310, K311}, \
    {K400, K401, K402, K403, K404, K405, K406, K407, K408, K409, K410, K411}  \
}
```

File: keyboards/mlego/m60/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
```
Comment (noroadsleft) [inline keyboards/mlego/m60/keymaps/default/keymap.c:19]:
```suggestion

enum layer_names {
    _QW = 0,
    _LWR,
    _RSE,
    _ADJ
};

```

File: keyboards/mlego/m60/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+
```
Comment (noroadsleft) [inline keyboards/mlego/m60/keymaps/via/keymap.c:19]:
```suggestion

enum layer_names {
    _QW = 0,
    _LWR,
    _RSE,
    _ADJ
};

```

File: keyboards/mlego/m60/m60.h [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+enum layer_names { _QW = 0, _LWR, _RSE, _ADJ };
+
```
Comment (noroadsleft) [inline keyboards/mlego/m60/m60.h:28]:
```suggestion
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**